conclusions/rq3.md

```diff
@@ -0,0 +1,5 @@
+# How are requirements established and used in software development?
+
+Section 2 describes the process of requirements engineering. Implementing structured requirements development and management early on can help deal with the challenges of building a large software systems. One of the main challenges is coordinating dependencies between requirements, and can be dealt with using coordination mechanisms.
+
+Additionally, the requirements engineering process should be kept iterative. The Twin Peaks model facilitates this and can be extended to work in conjunction with software architecture and design processes. This makes it possible to develop software modularly while still being able to verify the software meets its requirements.
```

Comment:
Change to `software meets its requirements` ==> `software meets the set requirements`?